### PR TITLE
[11.0][NEW] website_sale_stock_available_display: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Travis is red. Please indicate in our internal task tracker the link to this PR and also the module name.

**Comment by sergio-teruel** [comment]
> 2019-05-15 09:09:05,916 6460 [1;32m[1;49mINFO[0m openerp_test odoo.addons.website_sale_stock_available_display.tests.test_website_sale_stock_available_display: Ran 1 test in 15.366s
2019-05-15 09:09:05,917 6460 [1;32m[1;49mINFO[0m openerp_test odoo.addons.website_sale_stock_available_display.tests.test_website_sale_stock_available_display: OK
2019-05-15 09:09:05,917 6460 [1;32m[1;49mINFO[0m openerp_test odoo.modules.module: odoo.addons.website_sale_stock_available_display.tests.test_website_sale_stock_available_display tested in 15.37s, 4634 queries

Test for this module OK

File: website_sale_stock_available_display/models/res_config_settings.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    inventory_availability = fields.Selection(selection_add=[
```
Comment (pedrobaeza) [inline website_sale_stock_available_display/models/res_config_settings.py:10]: Isn't this a related field and thus automatically populated the new value?

File: website_sale_stock_available_display/static/src/js/website_sale_stock_available_display.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+odoo.define('website_sale_stock_available_display.load', function (require) {
+    'use strict';
+    var ajax = require('web.ajax');
+    var core = require('web.core');
+    var website_sale_stock = require('website_sale_stock.website_sale');
+    var QWeb = core.qweb;
+
+    var load_xml = ajax.loadXML('/website_sale_stock_available_display/static/src/xml/website_sale_stock_product_availability.xml', QWeb);
```
Comment (pedrobaeza) [inline website_sale_stock_available_display/static/src/js/website_sale_stock_available_display.js:8]: Do you need to add this through AJAX? Or is this for doing it asynchronously?

Comment (sergio-teruel) [reply website_sale_stock_available_display/static/src/js/website_sale_stock_available_display.js:8]: Yes, it is for extend website_sale_stock.product_availability template

File: website_sale_stock_available_display/models/res_config_settings.py (hunk 1 above)
Comment (sergio-teruel) [reply website_sale_stock_available_display/models/res_config_settings.py:10]: I am extending this field
https://github.com/odoo/odoo/blob/87d7adaf16a0adc37c0772a28e6e736c5f0898e4/addons/website_sale_stock/models/res_config_settings.py#L10

Comment (pedrobaeza) [reply website_sale_stock_available_display/models/res_config_settings.py:10]: Oh, I would expect that Odoo would put there a related field for not needing to define 2 times the same selection. Very ugly...

**Review by Tardo — APPROVED** [review]
LGTM, tested on runbot

**Review by rafaelbn — APPROVED** [review]
Tested in runbot :+1:

**Pull request merged**